libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -994,10 +993,9 @@ bool AP_Mission::mission_cmd_to_mavlink_int(const AP_Mission::Mission_Command& c
 
     case MAV_CMD_NAV_LOITER_UNLIM:                      // MAV ID: 17
         copy_location = true;
+        packet.param3 = (float)cmd.p1;
```

Comment:
Yes it does. Unlimited has a radius, and we store it, why would we not return it?